AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/Model/keyrings.smithy

```diff
@@ -339,8 +339,11 @@ structure CreateAwsKmsHierarchicalKeyringInput {
     @javadoc("How many seconds the Branch Key material is allowed to be reused within the local cache before it is re-retrieved from Amazon DynamoDB and re-authenticated with AWS KMS.")
     ttlSeconds: PositiveLong,
 
-    @javadoc("Which type of local cache to use.")
+    @documentation("Sets the cache bound OR the type of cache scoped (created & bound) to the Hierarchical Keyring. By providing an already initialized 'Shared' cache, users can determine the scope of the cache. i.e: When it is garbage collected or if the cache is bound to other Cryptographic Material Providers. If any other type in CacheType is provided, the Hierarchical Keyring will initialize a cache of that type, to be used with only this Hierarchical Keyring. If not set, a DefaultCache is initialized to be used with only this Hierarchical Keyring with entryCapacity = 1000.")
```

Comment:
how come this is now using a documentation trait as opposed to the javadoc trait?

Reply:
This is on Robin's recommendation that we should use documentation trait instead of the javadoc trait. This doesn't change the working at all for now, but is done in hopes of getting documentation for all the runtimes in the future.